src/components/BulkEnrollmentPage/hooks.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/prefer-default-export */
+import { DataTableContext } from '@edx/paragon';
+import { useContext, useEffect } from 'react';
+import { setSelectedRowsAction } from './data/actions';
+
+export const setExportedTableInstance = ({ dispatch }) => {
```

Comment:
is this function better named something like, `setSelectedRowsFromTable` or however else it makes sense to call it based on what it's doing?